datawig/mxnet_input_symbols.py

```diff
@@ -151,11 +152,11 @@ class EmbeddingFeaturizer(Featurizer):
 
     def __init__(self,
                  field_name: str,
-                 vocab_size: int = 100,
```

Comment:
Same here

Reply:
resolving for brevity. this occurs many times throughout.